dbt/include/snowflake/macros/materializations/dynamic_table/materialization.sql

```diff
@@ -93,27 +92,31 @@
 {% endmacro %}
 
 
-{% macro dynamic_table_execute_no_op(target_relation) %}
+{% macro dynamic_table_execute_no_op(dynamic_table) %}
```

Comment:
`target_dynamic_table` be a good name perchance?

Reply:
I shied away from a qualifier like `target_` or `existing_` when there were not conflicting versions of the same object. In hindsight, this should probably be generic, which would then look like `target_relation_name`, and leave the calculation of the name up to the caller; e.g. in Redshift MVs, this would be `materialized_view.mv_name` since that's how Redshift refers to that attribute in their docs.